packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -117,17 +117,16 @@ export function registerLocalNetwork() {
   }
 }
 
-export function isNetwork(network: L1Network | L2Network) {
-  const chainId = network.chainID
-
+export function isNetwork(chainId: ChainId) {
   return {
     // L1
     isMainnet: chainId === ChainId.Mainnet,
     // L1 Testnets
     isRinkeby: chainId === ChainId.Rinkeby,
     isGoerli: chainId === ChainId.Goerli,
     // L2
-    isArbitrum: Boolean((network as any).isArbitrum),
+    isArbitrum:
+      chainId === ChainId.ArbitrumOne || chainId === ChainId.ArbitrumNova,
```

Comment:
This could also be Arbitrum Rinkeby or Arbitrum Goerli

Reply:
Good catch! Fixed.